exercises/practice/grade-school/lib/school.ex

```diff
@@ -5,24 +5,33 @@ defmodule School do
   Each student is in a grade.
   """
 
+  @type school :: :dummy_type
```

Comment:
Is this a convention we have established anywhere else?

I don't recall it myself and this might not be intuitive to student approaching the problem.

So I think 3 options:
- initial type is `any()`
- specify the underlying data structure
- don't use a defined type and just use `any()` in the typespec

Reply:
I think I have seen this but I can't find it, maybe it was a different track...
First option sounds good.